static/index.html

```diff
@@ -14,6 +14,27 @@
         <script src="http://code.jquery.com/jquery-1.11.3.min.js"></script>
         <script src="http://knockoutjs.com/downloads/knockout-3.3.0.js"></script>
 
+        <style>
```

Comment:
In your opinion is it worth having the site's CSS all in one file, or is this website small/simple enough that embedded CSS isn't a concern?